File: 기능구현/program_h인덱스.js
function solution(citations) {
  var answer = 0;
  for(let i=0; i<1001; i++){
      let cnt = 0;
      citations.forEach((x)=>{
          if(x>=i) cnt+=1;
      })
      if(cnt>=i) answer=i;
  }
  return answer;
}

File: 기능구현/program_h인덱스.py
def solution(citations):
    
    if len(citations)==1 and citations[0]>=1:
        return 1
    
    answer = 0
    for i in range(0,1001):
        cnt = 0
        for x in citations:
            if x>=i:
                cnt+=1
        if cnt>=i:
            answer=i
    return answer